### ISSUE TITLE
Add mobile support and change in color scheme

### ISSUE THREAD

**Issue by anantshri** [libreprojects/libreprojects#124]
Two issues as soon as i visited this website.

1) Doesn't support mobile screens at all.
2) color scheme makes it hard to work with. Its like background is dark and text is slightly less dark. why can't we keep it like whilte background black text or inverse.

**Comment by jancborchardt** [comment]
@anantshri yeah, good points. Are you interested in working on pull requests to fix parts of this? :)

**Comment by anantshri** [comment]
I may endup doing something but can't promise for sure. I have done simmilar promisses in past but never got around doing it hence a bit hesitent on commiting anything and the whole reason of making an issues so that others also know that something like this could be of use / help.

**Comment by jancborchardt** [comment]
Ok, I appreciate your honesty. :) It’s good though that you care about the project so it would be cool seeing you contribute.

**Comment by Benjamin-Dada** [comment]
Hi, has the issue been fixed?

**Comment by jancborchardt** [comment]
@Benjamin-Dada no, except that it half-works on mobile. You or @anantshri (or anyone else interested) is very welcome to propose a pull request to fix it. :)
